skimage/color/colorlabel.py

```diff
@@ -1,6 +1,8 @@
 import itertools
 
 import numpy as np
+from scipy.ndimage import measurements
+from scipy.ndimage import find_objects
```

Comment:
In the rest of the library we use `from scipy import ndimage as ndi`. I would prefer to use the same convention here, as it makes the code easier to follow once you are familiar with that convention.